### DISCUSSION TITLE
Redlib - Request for privacy focused front-end for reddit app

### DISCUSSION THREAD

**Discussion by hanism02** [runtipi/runtipi-appstore#3178] [Requests]
### Description

Opensource Reddit privacy focused front end. A fork of Libreddit.  Currently still functions.
From their website:

> 10-second pitch: Redlib is a private front-end like [Invidious](https://github.com/iv-org/invidious) but for Reddit.

### Official Website

https://github.com/redlib-org/redlib

### Documentation link

https://github.com/redlib-org/redlib?tab=readme-ov-file#redlib

### Last application release & date

v0.31.2 (March 4, 2024)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/redlib-org/redlib

### Docker image link

https://quay.io/repository/redlib/redlib

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!
If you want a privacy focused front-end for Reddit, please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by jasonpearce** [comment]
This is a duplicate request of https://github.com/runtipi/runtipi-appstore/discussions/3816. Or rather, 3816 is a duplicate of 3178. I recommend consolidating to a single request.
